teaching_platform/users/db_handler.py

```diff
@@ -8,14 +8,17 @@
 from teaching_platform import validation
 
 
-def add_user(username, password):
+def add_user(username, password, email):
     validation.validate_username(username)
     encoded_password = validation.validate_password(password)
 
-    user = User(username=username, password=encoded_password, role=Role.STUDENT)
+    user = User(username=username, password=encoded_password, email=email, role=Role.UNCOMFIRMED)
 
     results = User.query.filter_by(username=username).all()
```

Comment:
out of curiosity - can you try something like the solution below? I don't know if that works

```
entry_exists = lambda kwarg: User.query.filter_by(**kwarg).all()

if entry_exists({"username": username}):
        if results: raise exceptions.UsernameTakenError()

if entry_exists({"email": email}):
        if results: raise exceptions.EmailTakenError()
```

Reply:
For now im not gonna implement this.